src/AccessibilityTranslator.js

```diff
@@ -51,18 +53,29 @@ export default class AccessibilityTranslator extends React.Component {
     return this._root;
   }
 
+  /**
+   * Starts the update of the accessibility DOM.  This should be called each time
+    the accessibility information for all DisplayObjects has been completed(e.g. just after
+    drawing a frame) to make sure that the canvas and accessibility DOM are in sync.
+   */
+  update() {
+    this.forceUpdate();
+  }
+
   _processDisplayObject(displayObject, parentBoundsInGlobalSpace) {
     if (!displayObject.accessible) {
       return;
     }
-
-    const role = displayObject.accessible.role;
+    let label; // eslint-disable-line
```

Comment:
Please specify which rule(s) needed to be disabled instead of disabling all linting.

Reply:
updated